src/lib/image/IOjp2/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ TARGET_INCLUDE_DIRECTORIES(
 
 TARGET_LINK_LIBRARIES(
   ${_target}
-  PUBLIC TwkFB openjpeg
+  PUBLIC TwkFB OpenJpeg::OpenJpeg
```

Comment:
Can we get rid of both src/lib/image/IOjp2 + src/lib/image/io_jp2 directories
now that the JPEG-2000 files are handled by OpenImageIO ?